### PR TITLE
UPSTREAM: clear ANSI color codes after tailing logs

### PR DESCRIPTION
**Pull request by fabianofranz**
Fixes https://github.com/openshift/origin/issues/5184

### CODE REVIEW

File: Godeps/_workspace/src/k8s.io/kubernetes/pkg/kubectl/cmd/log.go [hunk 1]
```diff
@@ -201,5 +205,22 @@ func (o LogsOptions) RunLog() (int64, error) {
...
+		inFd := file.Fd()
+		if term.IsTerminal(inFd) {
+			defer fmt.Fprint(o.Out, "\033[0m")
+			ch := make(chan os.Signal, 1)
+			signal.Notify(ch, syscall.SIGINT, syscall.SIGTERM, syscall.SIGQUIT)
+			defer signal.Stop(ch)
+			go func() {
+				if _, ok := <-ch; !ok {
+					return
+				}
```
Comment (smarterclayton) [inline Godeps/_workspace/src/k8s.io/kubernetes/pkg/kubectl/cmd/log.go]: It's probably cleaner to wrap out with something that interprets close and writes the clear, if a terminal is in use.  However, what commands do this today?  Does ssh clear the terminal state?  What about tail?

Comment (fabianofranz) [inline Godeps/_workspace/src/k8s.io/kubernetes/pkg/kubectl/cmd/log.go]: Actually this looks bad. Let me think about something better.

Comment (smarterclayton) [inline Godeps/_workspace/src/k8s.io/kubernetes/pkg/kubectl/cmd/log.go]: Post 3.1

Comment (fabianofranz) [inline Godeps/_workspace/src/k8s.io/kubernetes/pkg/kubectl/cmd/log.go]: Sure

**Comment by soltysh** [comment]
Please stick to commit [naming conventions](https://github.com/openshift/origin/blob/master/HACKING.md#cherry-picking-an-upstream-commit-into-origin) (UPSTREAM: \<pr number\>), so it's easy during next rebase.

**Pull request closed without merging**